okhttp/src/test/java/okhttp3/internal/tls/CertificatePinnerChainValidationTest.java

```diff
@@ -262,7 +261,7 @@
 
   @Test public void unrelatedPinnedIntermediateCertificateInChain() throws Exception {
     // https://github.com/square/okhttp/issues/4729
-    assumeFalse(getJvmSpecVersion().matches("1[123]"));
+    platform.expectFailure(platformMatches(CONSCRYPT_PROPERTY), KeyStoreException.class);
```

Comment:
I still think this reads awkwardly. I'd prefer a try/catch and then an assertion rather than framework.